README.md

```diff
@@ -188,10 +188,11 @@ Learn more about using
 ## Badge
 
 Show the world you're using _Jest_ →
-[![tested with jest](https://img.shields.io/badge/tested_with-jest-99424f.svg)](https://github.com/facebook/jest)
+[![tested with jest](https://img.shields.io/badge/tested_with-jest-99424f.svg)](https://github.com/facebook/jest) [![jest](https://rawgit.com/facebook/jest/master/website/static/img/jest-badge.svg)](https://github.com/facebook/jest)
```

Comment:
Once deployed, it will be available at https://facebook.github.io/jest/img/female-technologist/jest-badge.svg, which would be better than rawgit, I think

Reply:
Perhaps facebook.github.io maybe better, as I found rawgit.com has already been banned from accessing in CN.